docs/README-EDITING.md

```diff
@@ -0,0 +1,174 @@
+rm -rf ./<semver.version>
+```
+
+## Markdown containers
+
+To render content in different ways, the documentation uses custom Markdown containers, for example:
+
+```markdown
+::: example #exampleId .class :preset --html 1 --js 2
+
+// code example
+
+:::
+```
+
+We use the following Markdown containers:
+
+| Container                    | Usage                                                      |
+|------------------------------|------------------------------------------------------------|
+| `::: tip`                    | A note.                                                    |
+| `::: example [options]`      | Renders a code example as specified in [options].          |
+| `::: source-code-link [URL]` | Adds a source code link to the right of an API ref header. |
+
+### Adding code examples
+Using the `example` Markdown container, you can add code snippets that show the code's result:
+
+```js
+::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
+// `#example1` is the example's ID, for creating a Handsontable container
+    ```html
+    <div id="example1"></div>
+    ```
+    ```js
+    // code here
+    ```
+    ```css
+    /* custom css here */
+    ```
+:::
+```
+
+The `example` Markdown container offers the following options:
+
+| Option         | Required | Example         | Possible values                                            | Usage                                                                                              |
+|----------------|----------|-----------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `#exampleId`   | Yes      | `#data-grid-1`  | String                                                     | Container's unique ID.                                                                             |
+| `.class`       | No       | `.new-class`    | String                                                     | Container's custom class.                                                                          |
+| `:preset`      | No       | `:react`        | [`preset`](.vuepress/handsontable-manager/dependencies.js) | Sets code dependencies.                                                                            |
+| `--js <pos>`   | No       | `--js 1`        | Positive integer<br>(default `1`)                          | Sets the JS code snippet's position<br>in the markdown container.                                     |
+| `--html <pos>` | No       | `--html 2`      | Positive integer<br>(default `0`)                          | Sets the HTML code snippet's position<br>in the markdown container.<br><br>`0` disables the HTML tab. |
+| `--css <pos>`  | No       | `--css 2`       | Positive integer<br>(default `0`)                          | Sets the CSS code snippet's position<br>in the markdown container.<br><br>`0` disables the CSS tab.   |
```

Comment:
In case I have the following snippet:

<pre>
::: example #example1
```js
const container = document.querySelector('#example1');

const hot = new Handsontable(container, {
  data: Handsontable.helper.createSpreadsheetData(3, 11),
  colHeaders: true,
  rowHeaders: true,
  licenseKey: 'non-commercial-and-evaluation'
});
```
:::
</pre>

adding `--js 0` to `::: example #example1` guides to hang build/watch process.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik